playground/backend/pkg/environment/environment_service_test.go

```diff
@@ -0,0 +1,96 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I suppose that if it is a file with tests it should be placed into the `test` folder.

Reply:
nope, that doesn't GoLang way. We should place tests in the same place with the tested file